db/migrate/20220810093610_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] Please use the foreign key to author_id instead like I've shown in the posts above.

Reply:
thanks @zhadier